security/advancedtls/internal/testutils/testutils.go

```diff
@@ -35,12 +35,18 @@ type CertStore struct {
 	// ClientCert2 is the certificate sent by client to prove its identity.
 	// It is trusted by ServerTrust2.
 	ClientCert2 tls.Certificate
+	// ClientCert3 is the certificate sent by client to prove its identity.
+	// It is trusted by ServerTrust3. Used in CRL tests
+	ClientCert3 tls.Certificate
```

Comment:
At what point do we use an array instead of enumerating each one in its own variable?

Reply:
This is a great q. In general I find 'testutils.go' a little bit confusing - we load everything there and then use variables named like `cs.ServerCert1` and `cs.ClientCert1` across the codebase. So when I open a test it's tough to understand what exactly the certs mean.
Since you were the approver of [AuditLogger test](https://github.com/grpc/grpc-go/pull/6304) can I use the same pattern (test has properly named helper methods which load needed certs for it) instead?
https://github.com/grpc/grpc-go/blob/e88e8498c6df39703d1992bc4b677e7ef477145d/authz/audit/audit_logging_test.go#L329

Reply:
I'm fine with whatever.  Even this is acceptable.  I'm just trying to point out things I see that don't look as good as they could be.